JavaAssignments/src/amarja/Firstdiffindex.java

```diff
@@ -0,0 +1,42 @@
+package amarja;
+import java.util.Scanner;
+//Question 8 Program to find first different index from given array
+public class Firstdiffindex {
+		public static void main(String args[])
+		{		Scanner sc=new Scanner(System.in);//Reading the size and values of array 1
+				System.out.println("Enter size of first array:");
+				int size1=sc.nextInt();
+				System.out.println("Enter values of first array:");
+				int[]num1 = new int[size1];
+				for (int index=0;index<size1;index++)
```

Comment:
As discussed above , write separate function to take data from user.

Reply:
Incorporated review comments as suggested